### PR TITLE
feat: apply basic airbnb-base eslint rules

### PR DESCRIPTION
**Pull request by AhyoungRyu**
### Description Of Changes
Applies [UIKIT-3632](https://sendbird.atlassian.net/browse/UIKIT-3632)

Since we haven't had many rules in eslintrc.json which controls TS files, I thought we have more room to make our code cleaner and consistent. 
So as a first step, I applied `eslint-config-airbnb-base`(contains general JS code related rules only) which is a subset of `eslint-config-airbnb` (contains more rules for React than base one). I'll replace `airbnb-base` with `airbnb` later once we're ready.

I didn't want to change so many changes in this one PR so just fixed the 5 simple things
 - forced to use double quotes -> single quote 
 - removed redundant spaces
 - added trailing commas 
 - fixed wrong indentation
 - removed duplicated imports

and disabled other rules for now but will enable one-by-one and create a PR for each.

[UIKIT-3632]: https://sendbird.atlassian.net/browse/UIKIT-3632?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by AhyoungRyu** [comment]
https://sendbird.slack.com/archives/C050Z6DA70A/p1681363379887929

**Review by HoonBaek — APPROVED** [review]
LGTM, thanks for awesome work @AhyoungRyu !

File: package.json [hunk 1]
```diff
@@ -46,8 +46,9 @@
     "storybook": "start-storybook -p 6006",
     "build-storybook": "build-storybook",
     "lint": "yarn run eslint; yarn run eslint-ts",
-    "eslint": "eslint --no-eslintrc  -c .eslintrc.js 'src/**'",
-    "eslint-ts": "eslint --no-eslintrc -c .eslintrc.json './src/**/*.ts*'",
+    "lint:fix": "yarn run eslint --fix; yarn run eslint-ts --fix",
```
Comment (AhyoungRyu) [inline package.json:49]: Use this cmd before you push or commit. It'll automatically fix some fixable errors. 

Also, we can apply [husky pre-commit](https://github.com/typicode/husky#usage) as well later if we're all good with :)

**Review by sravan-s — APPROVED** [review]

**Comment by AhyoungRyu** [comment]
@bang9 Yeah I also want to merge those two files if it's possible. Thank you for the comment! 
Let me do this separately :)

**Pull request merged**